test/translator/read_capacity_10_test.cc

```diff
@@ -164,10 +162,9 @@ TEST_F(ReadCapacity10Test, BlocklengthLimit) {
             translator::ReadCapacity10ToScsi(buffer_, nvme_cmds_[0]));
   scsi::ReadCapacity10Data result = {};
   ASSERT_TRUE(translator::ReadValue(buffer_, result));
-  EXPECT_EQ(result.returned_logical_block_address,
-            identify_ns_.nsze > 0xffffffff ? 0xffffffff : identify_ns_.nsze);
+  EXPECT_EQ(result.returned_logical_block_address, 0);
   EXPECT_EQ(result.block_length,
-            identify_ns_.lbaf[identify_ns_.flbas.format].lbads);
+            __bswap_32(identify_ns_.lbaf[identify_ns_.flbas.format].lbads));
```

Comment:
is this compatible w both le, be?

Reply:
this currently is. but i'll change to using htonl and htoll for more clarify